pkg/util/volume_test.go

```diff
@@ -0,0 +1,162 @@
+/*
+Package util provides utility functions for OSD servers and drivers.
+Copyright 2017 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package util
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/libopenstorage/openstorage/volume/drivers/mock"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVolumeFromNameFailedToLocateDueToTooManyVolumes(t *testing.T) {
```

Comment:
should we add tests for inspect returning more than one volume or zero volumes (without error), although it does not happen?
similarly one for enumerate returning zero volumes without error?

Reply:
yeah, good point. I will add.